src/content/docs/apm/agents/java-agent/api-guides/guide-using-java-agent-api.mdx

```diff
@@ -37,7 +37,15 @@ For all available New Relic APIs, see [Intro to APIs](/docs/apis/getting-started
 
 ## Use the API [#api]
 
-To access the API class, add `newrelic-api.jar` to your application class path. The jar is in the New Relic Java agent's installation `zip` file. You can call the API when the Java agent is not running. The API methods are just stubs; the implementation is added when the Java agent loads the class.
+To access the API, download it from maven central via a build tool.
+
```

Comment:
@sdaubin The API page also asked for us to use the api jar. This can be confusing for customers so I decided to reference using build tools there as well.

Reply:
@obenkenobi I'm not sure why our documentation recommends the zip file so often.  It makes upgrading unnecessarily complicated.